domain/src/test/java/org/oppia/android/domain/oppialogger/loguploader/LogReportWorkManagerInitializerTest.kt

```diff
@@ -148,11 +180,59 @@ class LogUploadWorkManagerInitializerTest {
       )
       .build()
 
-    assertThat(logUploadWorkManagerInitializer.getWorkRequestDataForExceptions()).isEqualTo(
+    assertThat(logReportWorkManagerInitializer.getWorkRequestDataForExceptions()).isEqualTo(
       workerCaseForUploadingExceptions
     )
   }
 
+  @Test
+  fun testWorkRequest_verifyWorkRequestData_forSchedulingStorageUsageMetricLogs() {
+    val workerCaseForSchedulingStorageUsageMetricLogs: Data = Data.Builder()
+      .putString(
+        MetricLogSchedulingWorker.WORKER_CASE_KEY,
+        MetricLogSchedulingWorker.STORAGE_USAGE_WORKER
+      )
+      .build()
+
+    assertThat(
+      logReportWorkManagerInitializer.getWorkRequestDataForSchedulingStorageUsageMetricLogs()
+    ).isEqualTo(
+      workerCaseForSchedulingStorageUsageMetricLogs
+    )
```

Comment:
```suggestion
    ).isEqualTo(workerCaseForSchedulingStorageUsageMetricLogs)
```

Can fit on one line. Ditto elsewhere.

Reply:
done.